metrics.go

```diff
@@ -103,6 +106,14 @@ func NewMetricsHandler(address string, log log.Logger, db *database.Database) *M
 			Name: "whatsapp_puppets_total",
 			Help: "Number of WhatsApp users bridged into Matrix",
 		}),
+		activePuppetCount: promauto.NewGauge(prometheus.GaugeOpts{
+			Name: "whatsapp_active_puppets_total",
+			Help: "Number of active WhatsApp users bridged into Matrix",
+		}),
+		bridgeBlocked: promauto.NewGauge(prometheus.GaugeOpts{
```

Comment:
Is there not a boolean type? Do we have to use a Gauge?

Reply:
Prometheus doesn't have the concept of a boolean. A gauge allows you to go up and down, so is the best fit.